grails-app/domain/com/unifina/domain/data/Stream.groovy

```diff
@@ -35,7 +36,8 @@ class Stream implements Comparable {
 	Boolean autoConfigure = true
 	// Historical data storage period (days)
 	Integer storageDays = DEFAULT_STORAGE_DAYS
-
+	// inactivityThresholdHours is the inactivity period for a stream in hours
+	Integer inactivityThresholdHours = DEFAULT_INACTIVITY_THRESHOLD_HOURS
```

Comment:
- Should add to `constraints`:

`inactivityThresholdHours(nullable: false, min: 0)`

- Might trigger changes in `db-diff`, probably not